core/src/test/resources/log4j.xml

```diff
@@ -49,7 +49,7 @@
 
   <appender name="PENTAHOCONSOLE" class="org.apache.log4j.ConsoleAppender">
     <param name="Target" value="System.out"/>
-    <param name="Threshold" value="INFO"/>
+    <param name="Threshold" value="DEBUG"/>
```

Comment:
I believe this will negatively impact the performance of the tests. There is a programmatic way to set the logging level for a class:
import org.apache.log4j.Level;
import org.apache.log4j.LogManger;
... snip ...
Level lvl = Level.toLevel( "DEBUG" );
Class myTestClazz = TestClass.getClass();
LogManager.getLogger( myTestClazz ).setLevel( lvl );

Stick that code (or code like it) into the before action, and reset it in the after please.